### ISSUE TITLE
How to reproduce the results in the paper?

### ISSUE THREAD

**Issue by mrbeann** [DeqingYang/CKBC#1]
Hi, thank you for your excellent work. I tried to reproduce the results of the paper. but it seems to be missing lots of files, e.g., `data_bert/*`. Can you provide the missing files and give a guideline for reproducing the results? Thanks again.

**Comment by juju1103** [comment]
I'm sorry to see your question only now, I have received your feedback, I will update the documentation in the next two days to provide a guide that can be run, sorry for the inconvenience caused to you

**Comment by mrbeann** [comment]
Thank you for your reply. Looking forward to the update.

**Comment by DeqingYang** [comment]
Hi, I have re-updated the link address in the Dataset, you need to download it to the specified folder. Then, the codes I also updated, you just need to run python code/SIM_BERT_RGAT_ConvKB.py on it, you can try again when you have time. If you have any questions, please comment here.

**Comment by mrbeann** [comment]
Thank you for your reply. I've tried it but it seems the files in `data_bert` are still missing. E.g., vocab.txt file, config.json. (not sure if there are more)

**Comment by DeqingYang** [comment]
<img width="624" alt="image" src="https://user-images.githubusercontent.com/36389718/201524262-0852288c-4d8d-4e83-9863-f74c7165505a.png">
Don't worry about this file, it doesn't work inside. At this step the function returns directly and does not go to data_bert, so you can leave it alone.

**Comment by mrbeann** [comment]
Thank you for your prompt response. The file is ready to run. But it seems the value of the hyperparameter is not very good. Can you please check if there is something wrong with these values?
Here is the result of my run
```
Averaged stats for replacing head are ->
Hits@100 are 0.8668478260869565
Hits@10 are 0.6422101449275363
Hits@3 are 0.447463768115942
Hits@1 are 0.26358695652173914
Mean rank 166.61141304347825
Mean Reciprocal Rank 0.3863267074671392

Averaged stats for replacing tail are ->
Hits@100 are 0.8396739130434783
Hits@10 are 0.6684782608695652
Hits@3 are 0.4936594202898551
Hits@1 are 0.28442028985507245
Mean rank 473.0688405797101
Mean Reciprocal Rank 0.4182720479930321

Cumulative stats are ->
Hits@100 are 0.8532608695652174
Hits@10 are 0.6553442028985508
Hits@3 are 0.47056159420289856
Hits@1 are 0.27400362318840576
Mean rank 319.8401268115942
Mean Reciprocal Rank 0.4022993777300856

```

Also, I see that the baseline from the paper seems to be included here as well, and it would be great to have one on how to get the values in Table 1. (BTW, some other files seem wrong, e.g., `BERT_RGAT_ConvKB.py`)

**Comment by mrbeann** [comment]
Thank you for your previous efforts. Will this issue be updated?

**Comment by juju1103** [comment]
I obtained the baseline code mentioned in the paper and then changed it from their respective papers. I am sorry that I have cleared the specific code file.
Because of the novel coronavirus, I did not read the email, please understand if the reply is slow, I will update the code as soon as possible.

**Comment by juju1103** [comment]
<img width="510" alt="image" src="https://user-images.githubusercontent.com/58260042/204138214-ad30c2ce-7d00-406e-8e44-a60cea1fb2d9.png">

You can change this place and try again. Also, BERT_RGAT_ConvKB.py can you tell me what's wrong with that?

**Comment by mrbeann** [comment]
> <img alt="image" width="510" src="https://user-images.githubusercontent.com/58260042/204138214-ad30c2ce-7d00-406e-8e44-a60cea1fb2d9.png">
> 
> You can change this place and try again. Also, BERT_RGAT_ConvKB.py can you tell me what's wrong with that?

1. What do you mean by changing this and trying again? Can you provide the parameters that you used while running the experiments for the paper? 
2. As for BERT_RGAT_ConvKB, if you run it you can find that the model parameters do not match.

**Comment by Alijin** [comment]
> Thank you for your prompt response. The file is ready to run. But it seems the value of the hyperparameter is not very good. Can you please check if there is something wrong with these values? Here is the result of my run
> 
> ```
> Averaged stats for replacing head are ->
> Hits@100 are 0.8668478260869565
> Hits@10 are 0.6422101449275363
> Hits@3 are 0.447463768115942
> Hits@1 are 0.26358695652173914
> Mean rank 166.61141304347825
> Mean Reciprocal Rank 0.3863267074671392
> 
> Averaged stats for replacing tail are ->
> Hits@100 are 0.8396739130434783
> Hits@10 are 0.6684782608695652
> Hits@3 are 0.4936594202898551
> Hits@1 are 0.28442028985507245
> Mean rank 473.0688405797101
> Mean Reciprocal Rank 0.4182720479930321
> 
> Cumulative stats are ->
> Hits@100 are 0.8532608695652174
> Hits@10 are 0.6553442028985508
> Hits@3 are 0.47056159420289856
> Hits@1 are 0.27400362318840576
> Mean rank 319.8401268115942
> Mean Reciprocal Rank 0.4022993777300856
> ```
> 
> Also, I see that the baseline from the paper seems to be included here as well, and it would be great to have one on how to get the values in Table 1. (BTW, some other files seem wrong, e.g., `BERT_RGAT_ConvKB.py`)

Hello, Did you meet the issue "runtimeerror: module must have its parameters and buffers on device cuda:0 (device_ids[0]) but found one of them on device: cpu" when run the project? How to solve it?

**Comment by mrbeann** [comment]
No, but it seems if you can run it still cannot reproduce the results
